stencil-library/src/components/subaccounts-list/subaccounts-list.tsx

```diff
@@ -0,0 +1,117 @@
+import { Component, Host, h, Prop, State, Event, EventEmitter, Watch } from '@stencil/core';
+import { SubAccount } from '../../api/SubAccount';
+import { MapSubAccountStatusToBadge } from '../../utils/utils';
+import { PagingInfo, pagingDefaults } from '../table/table-utils';
+import { Api, IApiResponseCollection } from '../../api';
+
+@Component({
+  tag: 'justifi-subaccounts-list',
+  styleUrl: 'subaccounts-list.css',
+  shadow: true,
+})
+export class SubaccountsList {
+  @Prop() accountId: string;
+  @Prop() authToken: string;
+  @State() subaccounts: SubAccount[] = [];
+  @State() loading: boolean = true;
+  @State() errorMessage: string;
+  @State() paging: PagingInfo = pagingDefaults;
+  @Event({
+    eventName: 'subaccount-row-clicked',
+    bubbles: true,
+  }) rowClicked: EventEmitter<SubAccount>;
+
+
+  @Watch('accountId')
+  @Watch('authToken')
+  updateOnPropChange() {
+    this.fetchData();
+  }
+
+  connectedCallback() {
+    this.fetchData();
+  }
+
+  onPageChange = (direction: string) => {
+    return () => {
+      this.fetchData(direction);
+    }
+  }
+
+  async fetchData(direction?: string): Promise<void> {
+    if (!this.accountId || !this.authToken) {
+      this.errorMessage = "Can not fetch any data without an AccountID and an AuthToken";
+      this.loading = false;
+      return;
+    }
+    this.loading = true;
+    const endpoint = `account/${this.accountId}/seller_accounts`;
+
+    const response: IApiResponseCollection<SubAccount[]> = await Api(this.authToken, 'http://localhost:3000').get(endpoint, {
```

Comment:
I think we will probably need to get this value `localhost:3000` from an `.env` file.

Reply:
Yes, good catch. Thanks Tiago I totally forgot about that. 

I think you're right, we should merge the other branch with the .env file before merging this one and that should make this an easy change on this branch once that is completed.